content/post/go-git-v4.md

```diff
@@ -0,0 +1,73 @@
+---
+author: campoy
+date: 2017-12-07
+title: "Announcing the latest go-git!"
+image: /post/go-git-v4/go-git.png
+description: "After a year of intense work, we're happy to announce the latest and best release of go-git ever. go-git v4 includes many new features, making it the most used and feature complete git library written in Go, and in use on production at companies like source{d} and keybase."
+categories: ["git", "announcement", "OSS"]
```

Comment:
We have no categories  index yet, so these categories won't be shown in the blog.
We use the categories to fetch the posts from the source{d} homepage, so it is needed you add the following one: `technical`

Reply:
Ok, I'll add `technical`. Should I file an issue to add support for other categories?
This blog post is not necessarily technical

Reply:
Yes @campoy it would be great if you could open an issue :smile:

The current categories "technical", "culture" are only used for displaying the latest blog post in our Landing (under its proper section)
![image](https://user-images.githubusercontent.com/2437584/33715271-40f0ecfc-db52-11e7-811b-8347b1cbdf96.png)